packages/instrumentation-socket.io/README.md

```diff
@@ -36,6 +36,14 @@ socket.io instrumentation has few options available to choose from. You can set
 | `emitHook` | `SocketIoHookFunction` | hook for adding custom attributes before socket.io emits the event |
 | `onHook` | `SocketIoHookFunction` | hook for adding custom attributes before the event listener (callback) is invoked |
 | `traceReserved` | `boolean` | set to true if you want to trace socket.io reserved events (see https://socket.io/docs/v4/emit-cheatsheet/#Reserved-events) |
+| `filterHttpTransport`| `HttpTransportInstrumentationConfig` | set if you want to filter out the HTTP traces when using HTTP polling as the transport (see options below)
+
+`HttpTransportInstrumentationConfig` has a few options available to choose from. You can set the following:
```

Comment:
unclear what's the purpose of those options, why would I want to pass them and why are they needed?

Reply:
I will add a section explaining what the issue is and how this solves it

Reply:
@nirsky  how about this text? 
> If you use opentelemetry-instrumentation-socket.io alongside instrumentation-http while socket.io is using HTTP polling as the transport method. You will see an HTTP span created as the parent of the socket.io span. 
To filter out those spans, we use HttpTransportInstrumentationConfig see options below.